### PR TITLE
react ui icons replace

### PR DESCRIPTION
**Pull request by ferreroltd**

### CODE REVIEW

File: packages/libs/react-ui/src/components/Form/NumberField/NumberField.tsx [hunk 1]
```diff
@@ -151,14 +151,14 @@ export function NumberFieldBase(
...
-            icon={<ChevronUp size="sm" className={iconClass} />}
+            icon={<MonoExpandLess size="sm" className={iconClass} />}
             variant="text"
             isCompact
             className={buttonClass}
             {...incrementButtonProps}
           />
           <Button
-            icon={<ChevronDown size="sm" className={iconClass} />}
+            icon={<MonoExpandMore className={iconClass} />}
```
Comment (ferreroltd) [inline packages/libs/react-ui/src/components/Form/NumberField/NumberField.tsx:161]: check style is broken

File: packages/libs/react-ui/src/components/NavHeader/NavHeader.stories.tsx [hunk 2]
```diff
@@ -109,16 +113,16 @@ export const Dynamic: IStory = {
             </NavHeaderLink>
           ))}
         </NavHeaderLinkList>
-        <NavHeaderButton icon={<SystemIcon.Account />} />
+        <NavHeaderButton icon={<MonoAccountCircle />} />
```
Comment (ferreroltd) [inline packages/libs/react-ui/src/components/NavHeader/NavHeader.stories.tsx:116]: color style is off

File: packages/libs/react-ui/src/components/Form/NumberField/NumberField.tsx (hunk 1 above)
Comment (r-mulder) [reply packages/libs/react-ui/src/components/Form/NumberField/NumberField.tsx:161]: We force the size in the new button component.

File: packages/libs/react-ui/src/components/Button/Button.stories.tsx [hunk 3]
```diff
@@ -68,6 +68,13 @@ const meta: Meta<IButtonProps> = {
         type: 'boolean',
       },
     },
+    avatarProps: {
+      description:
+        'Props for the avatar component which can be rendered instead of startIcon',
+      control: {
+        type: 'boolean',
+      },
+    },
```
Comment (r-mulder) [inline packages/libs/react-ui/src/components/Button/Button.stories.tsx:77]: This was removed.

**Review by r-mulder — APPROVED** [review]

**Pull request merged**